tests/test_writer.py

```diff
@@ -145,3 +158,20 @@ def test_filter_by_tags_logic_is_triggered(tags_feature_file):
         include_tags=["some-tag-no-scenario-or-feature-has"],
     )
     assert results is None
+
+
+def test_empty_feature(empty_feature):
+    results = writer.feature_to_rst(empty_feature, empty_feature.parent)
+    assert results is None
+
+def test_empty_non_feature(empty_non_feature):
+    results = writer.feature_to_rst(empty_non_feature, empty_non_feature.parent)
+    assert results is None
+
+def test_empty_non_feature(commented_feature):
```

Comment:
This test function name duplicates the one above it.
I didn't catch this in my read through, but when I pulled this PR and did a `run-tests` I noticed that there weren't enough new test results reported.
Back to @dsayling for a quick fix to rename this function.